ConsentViewController/Classes/GDPRConsentViewController.swift

```diff
@@ -57,6 +57,16 @@ typealias Meta = String
         }
     }
 
+    // Default Message/PM language set to English
+    var messageLanguage: MessageLanguage = .English
+
+    /// Set language to see the Message/PM in given langauge
+    public var overwriteUserLanguageTo = MessageLanguage.English {
```

Comment:
Why do we need a second property for message language?

Reply:
With new changes I have removed the second variable, added to have more readability